src/components/menu/menu.stories.mdx

```diff
@@ -457,15 +457,15 @@ In order to align text and icons within a submenu a `Box` component will need to
 
 ### Scrollable submenu
 
-A scrollable submenu can be added using the `ScrollableBlock` component. This can be used for all of the submenu items, or just a selection, as shown in `Menu Itme Four` below.
-Note that only one `ScrollableBlock` can be used within a single submenu.
+A scrollable submenu can be added using the `ScrollableBlock` component. This can be used for all of the submenu items, or for a child list of one menu item.
```

Comment:
I think it would be a good idea to expand on your points with some visual examples of correct and incorrect uses of `ScrollableBlock` in the docs, similar to the examples you used in your tests. Perhaps something like this?:

```
Correct ✅ 
<Menu>
  <MenuItem submenu="Shopping items">
     <MenuItem>Apple</MenuItem>
     <MenuItem>Carrot</MenuItem>
     <MenuItem>
         Meat
         <ScrollableBlock>
             <MenuItem>Beef</MenuItem>
             <MenuItem>Chicken</MenuItem>
         </ScrollableBlock>
     </MenuItem>
  </MenuItem>
</Menu>

Incorrect ❌ 
<Menu>
  <MenuItem submenu="Shopping Items">
     <MenuItem>Apple</MenuItem>
     <MenuItem>Carrot</MenuItem>
     <MenuItem>Meat</MenuItem>
     <ScrollableBlock>
         <MenuItem>Beef</MenuItem>
         <MenuItem>Chicken</MenuItem>
     </ScrollableBlock>
  </MenuItem>
</Menu>
```